### DISCUSSION TITLE
Library maintenance status?

### DISCUSSION THREAD

**Discussion by PerWiklander** [com-lihaoyi/scalatags#273] [General]
I've been gone from Scalatags for a while (seven years or so) but would like to use it now for an experiment with [htmx](https://htmx.org/).

I'm wondering about the general maintenance status of the library. I know that scala tags is pretty feature complete but some of the issues and pull requests makes me wonder if the maintainers have other priorities and if that would make it harder to use / contribute to.

Some things I found along the way:

### Scala 3 related issues
- [Any reason to still allow unit fragments in scala 3? (2022-11-28)](https://github.com/com-lihaoyi/scalatags/issues/249)
- [Access modifiers in Scala 3 (2021-12-17)](https://github.com/com-lihaoyi/scalatags/issues/233)
- [Example doesn't work in Scala 3 with latest versions and sbt (2021-12-14)](https://github.com/com-lihaoyi/scalatags/issues/231)
This one has a merged PR, but is it fixed?

### Issues that are questions and could be moved here to discussions or the docs
 - [How to select tags in a stylesheet](https://github.com/com-lihaoyi/scalatags/issues/237)
 - [Scalatags only supports immutable Seq?](https://github.com/com-lihaoyi/scalatags/issues/236)

### PRs that should be merged or declined
- [feat: add in some missing Selectors #258 (2022-12-26)](https://github.com/com-lihaoyi/scalatags/pull/258)
- [Remove augment string #234 (2021-12-17)](https://github.com/com-lihaoyi/scalatags/pull/234)
- [Introduce apply as alternative to := #230 (2021-12-10)](https://github.com/com-lihaoyi/scalatags/pull/230)
- [Implemented pretty-printing for the text back-end. #127 (2016-07-02)](https://github.com/com-lihaoyi/scalatags/pull/127)
- [Fix js.map file #120 (2016-04-30)](https://github.com/com-lihaoyi/scalatags/pull/120)
- [Distinguish SVG attributes that do not overlap with HTML attributes #90 (2015-05-30)](https://github.com/com-lihaoyi/scalatags/pull/90)
- [Issue #76 Native pretty print (2015!)](https://github.com/com-lihaoyi/scalatags/pull/80)

And then a bunch of seven year old issues where it is unclear (to me) if they are still valid or if they have been fixed by the sands of time.

**Comment by lihaoyi** [comment]
From my perspective it's pretty feature complete. The test suite still passes. I still use it regularly, though admittedly my usage pattern does not cover every possible usage pattern.

I admit I could be more active in responding to PRs and issues. There's certainly existing rough edges (e.g. the entire CSS workflow), as well as potential places the library could "go" in future if someone had the vision for it.

If anyone wants commit/maintainer access, I'd be happy to give it out to anyone who will take it. It's not a particularly large or active repo and so the maintenance burden would not be large.
